src/QMCWaveFunctions/lcao/LCAOrbitalBuilder.h

```diff
@@ -57,6 +57,9 @@ class LCAOrbitalBuilder : public SPOSetBuilder
   std::string h5_path;
   ///Number of periodic Images for Orbital evaluation
   TinyVector<int, 3> PBCImages;
+  ///Phase Factor. Computed only once.
+  std::vector<ValueType> PhaseFactor;
```

Comment:
Since we got confused a couple of times, it is better to call it PeriodicImagePhaseFactors. Please update the comment line accordingly

Reply:
Done